Workflow in Windows started to fail

When the workflow tries to restart the docker service it now fails on GitHub Actions:

```
Run restart-service *docker*
  restart-service *docker*
  shell: C:\Program Files\PowerShell\7\pwsh.EXE -command ". '{0}'"
  env:
    DOCKER_HOST: tcp://localhost:2375
    JAVA_HOME_8.0.345_x64: C:\hostedtoolcache\windows\jdk\8.0.345\x64
    JAVA_HOME: C:\hostedtoolcache\windows\jdk\8.0.345\x64
    JAVA_HOME_8_0_345_X64: C:\hostedtoolcache\windows\jdk\8.0.345\x64
  
Restart-Service: D:\a\_temp\a3a06675-f318-4a15-8635-c7058adc5cfe.ps1:2
Line |
   2 |  restart-service *docker*
     |  ~~~~~~~~~~~~~~~~~~~~~~~~
     | Failed to start service 'Docker Engine (docker)'.

Error: Process completed with exit code 1.
```

It does not depend on the new Windows server 2022 virtual environment. It also fails now with `windows-2019`

--- a/.github/workflows/maven-windows.yml
+++ b/.github/workflows/maven-windows.yml
@@ -10,7 +10,7 @@ on:
 
 jobs:
   build:
-    runs-on: windows-2019
+    runs-on: windows-latest
     env:
       DOCKER_HOST: tcp://localhost:2375
 
@@ -27,12 +27,6 @@ jobs:
         path: ~/.m2
         key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml', '**/*.yml') }}
         restore-keys: ${{ runner.os }}-m2-
-    - name: Docker config
-      run: >
-        sc config docker
-        binpath="dockerd.exe --run-service -H tcp://localhost:2375"
-    - name: Docker restart
-      run: restart-service *docker*
     - name: Docker info
       run: docker info
     - name: Build with Maven

--- a/.github/workflows/maven-windows.yml
+++ b/.github/workflows/maven-windows.yml
@@ -10,7 +10,7 @@ on:
 
 jobs:
   build:
-    runs-on: windows-latest
+    runs-on: windows-2019
     env:
       DOCKER_HOST: tcp://localhost:2375
 

--- a/.github/workflows/maven-windows.yml
+++ b/.github/workflows/maven-windows.yml
@@ -7,6 +7,8 @@ on:
     branches:
       - master
   pull_request:
+  schedule:
+    - cron: "0 7 * * 0"
 
 jobs:
   build:

--- a/.github/workflows/maven-windows.yml
+++ b/.github/workflows/maven-windows.yml
@@ -27,8 +27,6 @@ jobs:
         path: ~/.m2
         key: ${{ runner.os }}-m2-${{ hashFiles('**/pom.xml', '**/*.yml') }}
         restore-keys: ${{ runner.os }}-m2-
-    - name: Docker info
-      run: docker info
     - name: Build with Maven
       run: mvn verify -Pdocker
       working-directory: com.examples.hellodocker

--- a/.github/workflows/maven-windows.yml
+++ b/.github/workflows/maven-windows.yml
@@ -11,8 +11,6 @@ on:
 jobs:
   build:
     runs-on: windows-latest
-    env:
-      DOCKER_HOST: tcp://localhost:2375
 
     name: Build in Windows
     steps:

--- a/.github/workflows/maven.yml
+++ b/.github/workflows/maven.yml
@@ -5,6 +5,8 @@ name: Java CI with Maven and Docker in Linux
 on:
   push:
   pull_request:
+  schedule:
+    - cron: "0 7 * * 0"
 
 jobs:
   build:

--- a/README.md
+++ b/README.md
@@ -7,3 +7,4 @@ An example of Chapter _Docker_ of the book
 by _Lorenzo Bettini_.
 
 See https://github.com/LorenzoBettini/tdd-buildautomation-ci-book-examples for the list of all the examples.
+

